tools/abel/validators/README.md

```diff
@@ -0,0 +1,192 @@
+
+### Entity Fields
+
+The `Entity Fields` tab defines fields translated by [reporting entities](../../../ontology/docs/building_config.md#reporting-physical-devices) and linked to [virtual entities](../../../onotology/docs/building_config.md#virtual-devices) defined in the `Entities` tab.
+
+`Entity Code` *string* **required**
+
+A [human-readable code](../../../onotology/docs/building_config.md#identifiers) for an entity which an entity field is a data point of. This code must already be defined in the [Entities](#entities) table. e.g. `AHU-1001`
+
+`Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) value for an entity corresponding to the entity in the Entity Code column ad is a unique [identifier for an entity](../../../onotology/docs/building_config.md#identifiers). This column need not be filled in unless the corresponding entity code is **not unique**. This column will be used when ABEL writes to a blank spreadsheet.
+
+`Reporting Entity Code` *string* **required**
+
+If a field is linked to a [virtual entity](../../../ontology/docs/building_config.md#virtual-devices) and its telemetry data is reported
+by some [reporting enity](../../../ontology/docs/building_config.md#reporting-physical-devices), then the `Entity Code` column must be the virtual entity code and the `reporting entity code` is the code of the
+entity reporting the field's data. This may be a gateway or some other IoT connected entity.
+
+`Reporting Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) [identifier](../../../ontology/docs/building_config.md#identifiers) for an entity correspong to the entity in the Reporting Entity Code Column. This value may be left blank unless the `reporting entity code` is **not unique**
+
+`DBO Standard Field Name` *string* **required**
+
+[Standardized field name](../../../ontology/docs/ontology.md#fields) belonging to an entity type as defined in the [Digital Buildings Ontology](../../../ontology). e.g. `zone_air_temperature_sensor`
+
+`Reporting Entity Field` *string*
+
+Name of a field as it is reported by a [gateway or other reporting entity](../../../onotology/docs/building_config.md#reporting-physical-devices). e.g. `zone_air_temperature_sensor_1` where the standard field name is `zone_air_temperature_sensor`. If the a field is not linked to a virtual entity then this value does not need to be filled in. Reporting entity fields may be [enumerated](../../../onotology/docs/ontology.md#enumeration).
+
+`Raw Field Path` *string* **required**
+
+Path to data point on a physical device's telemetry payload. The field path is used as the `present_value` in an entity type's [translation](../../../ontology/docs/building_config.md#defining-translations) to a physical device payload. e.g. `pointset.points.discharge_fan_run_command.present_value`.
```

Comment:
I dont think we have `pointset` it starts at points?

Reply:
Fixed.